signatures.go

```diff
@@ -20,17 +20,19 @@ import (
 	"crypto/sha256"
 	"encoding/binary"
 	"fmt"
-	"math"
+	"time"
 
 	"github.com/google/certificate-transparency-go/tls"
+	"github.com/google/certificate-transparency-go/x509"
 	"github.com/transparency-dev/formats/log"
-	"github.com/transparency-dev/static-ct/modules/dedup"
-	"golang.org/x/crypto/cryptobyte"
 	"golang.org/x/mod/sumdb/note"
 
 	ct "github.com/google/certificate-transparency-go"
 )
 
+const NanosPerMilli int64 = int64(time.Millisecond / time.Nanosecond)
```

Comment:
Did you mean to export it BTW?

Reply:
Woopsie, thanks. As a general comment, the repo needs a good clean-up of exported / unexported stuff as well. I don't want to do this straight away though first I'll re-organize and simplify the code, then I'll carve a few things into an internal/ directory, and then export / unexport accordingly.